CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **IMPORTANT**: scikit-bio is no longer compatible with Python 2. scikit-bio is compatible with Python 3.4 and later.
 
 ### Features
+* Added trailing whitespace when reading runon GF/GS lines with `skbio.io.format.stockholm` reader. ([#1328](https://github.com/biocore/scikit-bio/issues/1328))
```

Comment:
This is a bug fix so put under "Bug fixes". Since this was a bug, rephrase to describe what the previous behavior was, and what the new behavior is:

"""
Fixed bug when reading Stockholm files with GF or GS features split over multiple lines. Previously, the feature text was simply concatenated because it was assumed to have trailing whitespace. There are examples of Stockholm files with and without trailing whitespace for multi-line features, so the `skbio.io.format.stockholm` reader now adds a single space when concatenating feature text without trailing whitespace to avoid joining words together. Multi-line trees stored as GF metadata are concatenated as they appear in the file; a space is not added when concatenating.
"""